orchestrator/package.json

```diff
@@ -22,7 +45,7 @@
     "geolib": "^2.0.24",
     "ioredis": "^4.0.0",
     "joi": "^10.2.0",
-    "kubernetes-client": "^9.0.0",
+    "kubernetes-client": "8.3.6",
```

Comment:
Was there no change between 8 and 9 that could break?

Reply:
If I'm not wrong we were using version 5.x.x, which is very old. Someone updates the package.json, but not the package-lock.json. Versions greater than 8.3.6 requires newer versions of node. I think it would be necessary to retest in Kubernetes.